experimental/dds/tree2/src/shared-tree-core/editManager.ts

```diff
@@ -438,8 +444,10 @@ export class EditManager<
 		revision: RevisionTag,
 		change: TChangeset,
 		rebaseAnchors = true,
+		undoRedoType = UndoRedoManagerCommitType.Undoable,
 	): Delta.Root {
 		this.pushToLocalBranch(revision, change);
+		this.localCommitTypes.set(revision, undoRedoType);
```

Comment:
nit: I think this should go inside of `pushToLocalBranch`. If we ever were to push to the local branch by some other means in the future, we'd still want to update the map.

Reply:
The rollback case is weird though. Don't worry about it for now - it's going to change in my stuff anyway.